circuit-lib/circuitlib-rs/src/inclusion/merkle_inclusion_proof.rs

```diff
@@ -0,0 +1,44 @@
+use std::{collections::HashMap, fs::File};
+
+use ark_circom::{circom::Inputs, read_zkey};
+
+use crate::{
+    arkworks_prover::{prove, ArkProvingKey},
+    errors::CircuitsError,
+    inclusion::{
+        merkle_inclusion_proof_inputs::{InclusionMerkleProofInputs, InclusionProofInputs},
+        merkle_tree_info::MerkleTreeInfo,
+    },
+    prove_utils::ProofResult,
+};
+
+pub fn merkle_inclusion_proof(
```

Comment:
This function has hardcoded test files as inputs, please either rename the function to test_merkle_inclusion_proof
```suggestion
pub fn test_merkle_inclusion_proof(
```